qml/pages/CameraUI.qml

```diff
@@ -608,4 +629,26 @@ Page {
 
         return siz.width + "x" + siz.height;
     }
+
+    RoundButton {
+        id: btnGallery
+
+        visible: false
+        enabled: false
```

Comment:
These seem to be always set to the same value so maybe using "enabled: visible" and in other parts of the code only setting the visible value would make code a little simpler.